charts/trento-server/charts/trento-mcp-server/templates/ingress.yaml

```diff
@@ -0,0 +1,70 @@
+{{- /*
+Copyright 2025 SUSE LLC
+SPDX-License-Identifier: Apache-2.0
+*/}}
+{{- if .Values.ingress.enabled -}}
```

Comment:
I would do one thing.
I would put the default value of the ingress to `true` and here, create the ingress if the mcp server is enabled.
`{{- if .Values.enabled -}}`.
After all, many users (who don't usually use helm outside trento) will need the ingress if the mcp server is enabled.
And if they want other different ingress, they can disable manually